src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -13,6 +13,8 @@ import quizTemplate from '../../fixtures/quiz_exercise_fixtures/quizExercise_tem
 export const COURSE_BASE = BASE_API + 'courses/';
 export const COURSE_MANAGEMENT_BASE = BASE_API + 'course-management/';
 export const PROGRAMMING_EXERCISE_BASE = BASE_API + 'programming-exercises/';
+const QUIZ_EXERCISE_BASE = BASE_API + 'quiz-exercises/';
```

Comment:
Is there a reason why you don't export this? Even if you don't use it somewhere else I think in this context it makes sense to export it anyway to stay uniform

Reply:
Yeah it was added by the merge from develop. I agree, it makes sense to also export that constant